sycl/doc/extensions/deprecated/sycl_ext_oneapi_discard_queue_events.asciidoc

```diff
@@ -0,0 +1,211 @@
+= sycl_ext_oneapi_discard_queue_events
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+// This is necessary for asciidoc, but not for asciidoctor
+:cpp: C++
+
+== Introduction
+
+IMPORTANT: This specification is a draft.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are
+trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc.
+used by permission by Khronos.
+
+This document describes an extension that introduces a `discard_events` property for
+SYCL queues. This property enables developers to inform a SYCL implementation that
+the events returned from queue operations will not be used.
+
+== Notice
+
+Copyright (c) 2021 Intel Corporation. All rights reserved.
+
+== Status
+
+This extension has been deprecated.  Although the interfaces defined in this
+specification are still supported in {dpcpp}, we expect that they will be

```

Comment:
You need to define `dpcpp` as an asciidoc attribute at the top of the file.

Reply:
I saw you fixed the `dpcpp` in the other asciidoc file, but this one is not yet fixed.

Reply:
Done

Reply:
> I saw you fixed the `dpcpp` in the other asciidoc file, but this one is not yet fixed.

Does this line not fix it? https://github.com/intel/llvm/pull/18059/files#diff-2bb29c57c2d8ec658778edf583e9084912e6f13dbebe26508c7b2da012738158R11

It seems to render "DPC++" correctly.